src/LatticeInverseProblem/core.py

```diff
@@ -79,16 +79,47 @@ def generalised_ft(t, alpha, sigma, e, e0):
     res = mp.fmul(res, arg) # Erfc() * exp()
     return res
 
-def ft_mp(e, t, sigma_, alpha, e0=mpf("0"), type="EXP", T=0, ker_type='GAUSS'):
+def generalised_ft_halfnorm(t, alpha, sigma, e, e0):
```

Comment:
@LupoA, I introduced again the possibility of a half normalised gaussian to be consistent between the paper it will come out and the code release. At least in the version I release on Zenodo I would keep that. Then, it's fine to me to remove it. 

Thanks also for the suggestion.

Reply:
very good, let's indeed keep it for the release.

In order to use runExact.py, you should modify (locally on your machine) line 105 where you should put your kernel of choice. Then in line 152 you should tell the code to compute the coefficients with the same kernel. It should be a painless check, but let me know if I can assist!

Reply:
Since there will need to be a code release that wraps this library, could the normalisation be done there and this library kept "pure"? (Can discuss in more detail if my meaning isn't clear here.)

Reply:
> you should modify (locally on your machine) line 105 where you should put your kernel of choice. Then in line 152 you should tell the code to compute the coefficients with the same kernel

You have to modify the installed library in place? This sounds like a design issue that is out of scope for addressing in this PR but is worth trying to improve.

@LupoA am I reading you correctly? If so then we can open an issue to track this.

Reply:
not the library, runExact.py is supposed to be an example file or rather a template for a specific application (performing the reconstruction on synthetic, error-free data). The exact smeared spectral density will depend on the specific kernel.

It is of course possible to input the kernel and let runExact work on any of them, by adding a couple of if statement. If we want this, it should be part of the "additional kernel implementation" effort that is already inside this PR (see commit 'adding cauchy kernel')

Reply:
Ahh, I see. In that case the version of `runExact.py` or similar that Niccolò uses in our analysis should be in the code release for the paper, not the more general library.

Reply:
I am quite happy with what @nickforce98 has done having it in the example folder! notice that this specific file is not used in your analysis since you use real data, it is however an important file for checking the implementation is correct